templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
The link for the user profile uses `user.id` to generate the URL for `driver-detail`. Ensure that the `user` object has an `id` attribute and that the `driver-detail` URL pattern is correctly configured to accept this parameter.